### ISSUE TITLE
Firmware 1.0.11 TDSR12 ELEV2 not working under stabilization mode

### ISSUE THREAD

**Issue by eliasmoby** [FrSkyRC/ETHOS-Feedback-Community#3115]
The firmware 1.0.11 for TDSR12 not work correctly when you use double channel for elevators, when i downgrade to 1.0.6 works correctly

https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/124819212/5a824980-1fb3-427a-a4c8-79a7bed60a5a

https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/124819212/609ff63b-838e-463e-8d48-af0608d236ba

Regards in advance

**Comment by eliasmoby** [comment]
I did with newly fresh model and reload the last firmware and dont work
properly

Regards in advance

**Comment by eliasmoby** [comment]
Tks, i can solve it with your instructiosn, but one more question channel 8 works at rudder stab gyro, i only need 1 rudder in channel 4, so this channel 8 its unable to use with other source? Regards in advanceEnviado desde mi Samsung Mobile de Telcel
-------- Mensaje original --------De: mawzthefinn ***@***.***> Fecha: 3/12/2023  7:36 p. m.  (GMT-06:00) A: FrSkyRC/ETHOS-Feedback-Community ***@***.***> CC: eliasmoby ***@***.***>, Author ***@***.***> Asunto: Re: [FrSkyRC/ETHOS-Feedback-Community] Firmware 1.0.11 TDSR12 ELEV2 not working under stabilization mode (Issue #3115) 

I did with newly fresh model and reload the last firmware and dont work properly Regards in advance El dom,

A new model file and flashing the firmware won't solve the issue you demonstrated. Doing the full self-check procedure in the receiver manual however should solve it. The receiver stores the demonstrated endpoints when a self-check is done, and does not update until a new self-check is done. Even a firmware update will not change that.

Also.i have 4 aileron, 2 elevators, 1 ruder and 1 ch for engine how can i use the estabilization because luascript only show optios for ail2, ele2 in page 1 and page 2 ch10, ch11 said ail4 and ele4 what happend with ail3?

The stabilizer functions as two sets. AIL1, ELE1 and RUD1 are all enabled/disabled automatically when the stabilizer on page 1 is enabled/disabled (Stabilizer set 1), and AIL2 and ELE2 can be optionally enabled (they are part of Stabilizer Set 1).
Page 2 controls stabilizer set 2. AIL3, ELE2 and RUD2 are all enabled/disabled automatically when the stabilizer on page 2 is enabled/disabled (Stabilizer set 2), and AIL4 and ELE4 can be optionally enabled (they are all part of Stabilizer Set 2)

**Comment by mawzthefinn** [comment]
> Tks, i can solve it with your instructiosn, but one more question channel 8 works at rudder stab gyro, i only need 1 rudder in channel 4, so this channel 8 its unable to use with other source? 

Yes, you lose the ability to re-assign ch8 (ele3) and Ch9 (Rud2) for other uses when you enable Stabilizer Set 2 for Ail3(Ch7) and Ail4(Ch10)

Note you can assign other channels (say Ch16 & ch17) to Ports 8 and 9 on the receivers, so you can still connect up to 12 servos or devices to the receiver, it's only the RF channels sent from the radio that are dedicated to stabilizer functions.

**Comment by eliasmoby** [comment]
Ok, thanks for the clarification.
